fxdc/fields.py

```
@@ -0,0 +1,81 @@
+        self,
+        desc: Optional[str] = None,
+        verbose_name: Optional[str] = None,
+        default: Optional[Any] = None,
+        typechecking: bool = True,
+        null: bool = True,
+        blank: bool = True,
+    ):
+        if verbose_name:
+            if verbose_name[0] not in string.ascii_letters + "_":
+                raise FieldError("Name Should Start With A Letter")
+            if any([x not in ACCEPTABLECHARACTERS for x in verbose_name]):
```

Ruff (0.12.2)

Unnecessary list comprehension

Remove unnecessary comprehension

(C419)